content/guards.md

```diff
@@ -224,7 +233,16 @@ export class RolesGuard {
     }
     const request = context.switchToHttp().getRequest();
     const user = request.user;
-    return matchRoles(roles, user.roles);
+    return this.matchRoles(roles, user.roles);
+  }
+
+    private matchRoles(requiredRoles: string[], userRoles: string[]): boolean {  
```

Comment:
Doesn't look this code is formatted. Kindly fix the formatting and the logic as per my previous message.